test_runner/regress/test_layers_from_future.py

```diff
@@ -49,7 +49,7 @@ def test_issue_5878(neon_env_builder: NeonEnvBuilder):
         "compaction_period": "0s",  # we want to control when compaction runs
         "checkpoint_timeout": "24h",  # something we won't reach
         "checkpoint_distance": f"{50 * (1024**2)}",  # something we won't reach, we checkpoint manually
-        "image_creation_threshold": f"{image_creation_threshold}",
+        "image_creation_threshold": "100", # we want to control when image is created
```

Comment:
```suggestion
        "image_creation_threshold": "100",  # we want to control when image is created
```

To fix `check-codestyle-python` that will fail (it doesn't block test run, so I'd wait for the results before pushing the fix)

Reply:
Thank you. I've enabled pre-commit check for future. May be you have script to run fast tests locally?